.github/actions/install-dependencies/action.yml

```
@@ -61,6 +65,21 @@ runs:
         sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
         echo "::endgroup::"
 
+    - name: Install software for WebAssembly compilation
+      # Boolean inputs are actually strings:
+      # https://github.com/actions/runner/issues/1483
+      if: inputs.wasm-build == 'true'
+      shell: bash
+      run: |
+        echo "::group::Install software for WebAssembly compilation"
+        sudo apt-get update
+        git clone https://github.com/emscripten-core/emsdk.git
+        cd emsdk/
+        ./emsdk install '3.1.18'
+        ./emsdk activate '3.1.18'
+        echo "/home/runner/work/cvc5/cvc5/emsdk" >> $GITHUB_PATH
+        echo "/home/runner/work/cvc5/cvc5/emsdk/upstream/emscripten" >> $GITHUB_PATH
```

One suggestion to make the specified paths more resilient to changes:
```suggestion
        echo "$(pwd)" >> $GITHUB_PATH
        echo "$(pwd)/upstream/emscripten" >> $GITHUB_PATH
```